### PR TITLE
Cortx1.0 udx

### PR DESCRIPTION
**Pull request by Vrishali-web-dev**
UDX - UI workflow.
1. S3 account select/create
2. bucket creation/selection
3. I am user creation
4. Registration screen

### CODE REVIEW

File: gui/src/common/regex-helpers.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const udxBucketNameRegex = helpers.regex(
   /^[a-z0-9-]{0,51}[a-z0-9]$/
 );
 // tslint:disable-next-line
-export const udxBucketNameTooltipMessage = `To identify the S3 bucket correctly, the Bucket name will always use "udx" as its prefix. The bucket name must be of minimum 5 characters and maximum 56 characters.
+export const udxBucketNameTooltipMessage = `To identify the S3 bucket correctly, the Bucket name will always use "ldp" as its prefix. The bucket name must be of minimum 5 characters and maximum 56 characters.
```
Comment (kupranay) [inline gui/src/common/regex-helpers.ts:58]: I think we dont require this now as per our last conversation

File: gui/src/components/health/cortx-severity-based-health-view.vue [hunk 2]
```diff
@@ -16,15 +16,16 @@
 */
 <template>
   <div class="pa-5">
-    <div
-      class="cortx-back-btn"
-      @click="$router.go(-1)"
-    >
+    <div class="cortx-back-btn" @click="$router.go(-1)">
```
Comment (kupranay) [inline gui/src/components/health/cortx-severity-based-health-view.vue:19]: why do we require any change in this file?

File: gui/src/components/dashboard/dashboard.vue [hunk 3]
```diff
@@ -16,13 +16,25 @@
 */
 <template>
```
Comment (kupranay) [inline gui/src/components/dashboard/dashboard.vue:17]: why do we require any change in this file

File: gui/src/components/maintenance/cortx-about.vue [hunk 4]
```diff
@@ -17,9 +17,13 @@
 <template>
   <div class="cortx-p-1">
     <div class="cortx-text-lg cortx-text-bold" id="lblAbout">
-      {{$t("aboutUs.title")}}
+      {{ $t("aboutUs.title") }}
```
Comment (kupranay) [inline gui/src/components/maintenance/cortx-about.vue:20]: No changes in this file is required

File: gui/src/components/s3/access-key-management.vue [hunk 5]
```diff
@@ -175,7 +175,7 @@ export default class CortxAccessKeyManagement extends Vue {
     if (this.s3UrlNone) {
       delete this.accessKeyDetails["S3 URL"];
     }
-    
+
```
Comment (kupranay) [inline gui/src/components/s3/access-key-management.vue:178]: No changes in this file is required

File: gui/src/components/s3/bucket-creation.vue [hunk 6]
```diff
@@ -385,9 +385,9 @@ export default class CortxBucketCreation extends Vue {
         value: "name",
         sortable: false
       },
-      { 
-        text: "Action", 
-        value: "data-table-expand" 
+      {
```
Comment (kupranay) [inline gui/src/components/s3/bucket-creation.vue:388]: No changes in this file is required

File: gui/src/components/s3/iam-user-management.vue [hunk 7]
```diff
@@ -453,9 +453,9 @@ export default class CortxIAMUserManagement extends Vue {
         value: "arn",
         sortable: false
       },
-      { 
-        text: "Action", 
-        value: "data-table-expand" 
+      {
```
Comment (kupranay) [inline gui/src/components/s3/iam-user-management.vue:456]: No changes in this file is required

**Review by kupranay — APPROVED** [review]

File: gui/src/common/regex-helpers.ts (hunk 1 above)
Comment (Vrishali-web-dev) [reply gui/src/common/regex-helpers.ts]: Removed the constant

File: gui/src/components/dashboard/dashboard.vue (hunk 3 above)
Comment (Vrishali-web-dev) [reply gui/src/components/dashboard/dashboard.vue]: Reverted the formatting related changes

File: gui/src/components/s3/bucket-creation.vue (hunk 6 above)
Comment (Vrishali-web-dev) [reply gui/src/components/s3/bucket-creation.vue]: Resolved

File: gui/src/components/s3/iam-user-management.vue (hunk 7 above)
Comment (Vrishali-web-dev) [reply gui/src/components/s3/iam-user-management.vue]: Resolved

File: gui/src/components/s3/access-key-management.vue (hunk 5 above)
Comment (Vrishali-web-dev) [reply gui/src/components/s3/access-key-management.vue]: Resolved

File: gui/src/components/health/cortx-severity-based-health-view.vue (hunk 2 above)
Comment (Vrishali-web-dev) [reply gui/src/components/health/cortx-severity-based-health-view.vue]: Resolved

**Review by soniyamoholkar — APPROVED** [review]

**Review by pawankumarsrivastava — APPROVED** [review]

**Pull request merged**